### ISSUE TITLE
Is activating a tri-state toggle limited to toggling between aria-pressed=false and aria-pressed=true?

### ISSUE THREAD

**Issue by mcking65** [w3c/aria#684]
The [aria-pressed spec](https://w3c.github.io/aria/#aria-pressed)
includes the following language:

> Toggle buttons require a full press-and-release cycle to change their value. Activating it once changes the value to true, and activating it another time changes the value back to false. A value of mixed means that the values of more than one item controlled by the button do not all share the same value. Examples of mixed-state buttons are described in WAI-ARIA Authoring Practices [WAI-ARIA-PRACTICES-1.1].

This text implies that activating a tri-state toggle cannot result in `aria-pressed="mixed"`. Is this an intentional restriction?

We do not have such a limitation on the tri-state checkbox, which in some implementations, remembers the configuration of its most recent mixed state as demonstrated in the 
[APG tri-state checkbox example](http://w3c.github.io/aria-practices/examples/checkbox/checkbox-2/checkbox-2.html).

I do not see a reason for preventing tri-state toggles from having this mixed state activation feature of tri-state checkboxes. If there is no reason for such a limitation, I will propose alternative text.

**Comment by jnurthen** [comment]
The text "Examples of mixed-state buttons are described in WAI-ARIA Authoring Practices [WAI-ARIA-PRACTICES-1.1]." does not seem to be true and should be removed.
IMO we should also remove reference to mixed values on buttons. I can't see any circumstance where this would be useful. I'm happy for it to remain valid but calling it out in prose seems strange.

**Comment by jnurthen** [comment]
In windows an indeterminate toggle button has the same visual state as an unpressed toggle button
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.primitives.togglebut

MacOS allows a mixed state button but seems to discourage its usage
https://developer.apple.com/documentation/appkit/nsbutton?language=objc

**Comment by jnurthen** [comment]
closing as insufficient need.
